src/plugins/azgenerator/TemplateAzureCliParams.ts

```diff
@@ -171,31 +170,41 @@ function getCommandBody(model: CodeModelAz, needUpdate: boolean = false, needGen
 
                         argument = "        c.argument('" + parameterName + "'";
                         argument += ", options_list=['--" + parameterName.substr(0, parameterName.length - 1) + "']";
-                    } else if (parameterName.endsWith('name') && parameterName.replace(/_name|_/g, '') == model.CommandGroup_DefaultName.toLowerCase()) {
+                        model.MethodParameter.language['az']['alias'] = Array(parameterName.substr(0, parameterName.length - 1));
+                    } else if (parameterName.endsWith('name') && parameterName.replace(/_name|_/g, '') == model.CommandGroup_DefaultName.toLowerCase() || !isNullOrUndefined(model.MethodParameter?.language?.['cli']?.['alias'])) {
```

Comment:
>parameterName.replace(/_name|_/g, '') [](start = 65, length = 37)

_name should be at the end of string, right? this may replace things unexpected.

Reply:
Changed. please review it again. thx